Monika After Story/game/script-topics.rpy

```diff
@@ -7254,23 +7254,23 @@ init 5 python:
 
 label monika_solipsism:
     m 3eub "Have you ever heard about solipsism, [player]?"
-    m 3eua "It's a philosophical idea which states that only your mind is sure to exist."
-    m 1eud "What do you think of that?"
-    m "Is there really something other than yourself?"
-    m 1euc "Or maybe you and everything else are just some representations of my psyche?"
-    m 1dsd "Extensions of my own consciousness..."
-    m 1hub "Ahaha!"
-    m 1eub "I'm just kidding, of course."
-    m 1eua "It's more likely that {i}I'm{/i} a representation of your mind than the other way around."
-    m "I'm the 'game character', right?"
-    m 1dsc "Still...it makes you wonder about the nature of our realities. Or maybe 'fabric' would be a better term than 'nature'?"
-    m 1eua "Anyway, I recommend reading more about solipsism."
-    m 1eub "There are several varieties of it."
-    m "Like metaphysical, epistemological, or methodological solipsism."
-    m 1eua "You can read more about them online."
-    m "Then we could talk about these topics more in-depth."
-    m 1hub "I'm really looking forward to talking about philosophy with you, [player]!"
-    m 1eua "I want to know more about the way you think..."
+    m 3eua "It's an idea which states that only you yourself exists."
+    m 1eud "'Cogito, ergo sum.'"
+    m "'I think, therefore I am.'"
+    m "If you're sure your self exists, how about others outside your self?"
+    m 1euc "Maybe everyone else is just a figment of our imagination, [player]."
+    m "Maybe in reality, we're the only consciousness in this world in a vast sea of fake minds..."
+    m 1dsd "Creations of our wild machinations..."
+    m 1eub "Ahaha, I'm just kidding~"
+    m 1eud "I do believe we can trust our own existences and doubt others' their own..."
+    m 3eua "But at the same time, we can't really disprove theirs, can we?"
```

Comment:
```suggestion
    m 3eua "But at the same time, we can't really disprove their's, can we?"
```

Reply:
This is grammatically incorrect, Wick. If you were trying to make "theirs" singular it wouldn't make sense anyways since she's referring to multiple people in the previous sentence.

Reply:
He's actually right, you had it wrong. It shouldn't be used possessively